relays/ethereum/src/headers.rs

```diff
@@ -411,6 +587,14 @@ fn remove_header<P: HeadersSyncPipeline>(
 	header
 }
 
+/// Get header from the qeueue.
+fn header<'a, P: HeadersSyncPipeline>(
```

Comment:
```suggestion
fn get_header_from_queue<'a, P: HeadersSyncPipeline>(
```

Reply:
I'm suggesting this since when I was reading some of the code that was like `HeaderStatus::MaybeOrphan => header(...)` it wasn't clear at first glance that `header()` was fetching stuff from the queues in the same way that seeing `queue.get(id)` would be.